<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="dell-PC" ignored="3" name="Test" tests="8" failures="7" timestamp="10 Mar 2018 11:17:49 GMT" time="108.973" errors="0">
  <testcase name="loginAndLogout" time="4.093" classname="testcases.TC002_LoginForNegative">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.click(SeMethods.java:135)
	at pages.LoginPage.clickLogInForFailer(LoginPage.java:48)
	at testcases.TC002_LoginForNegative.loginAndLogout(TC002_LoginForNegative.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="16.563" classname="testcases.TC001_LoginAndLogout">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.verifyPartialText(SeMethods.java:240)
	at pages.HomePage.verifyLoggedName(HomePage.java:30)
	at testcases.TC001_LoginAndLogout.loginAndLogout(TC001_LoginAndLogout.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="4.035" classname="testcases.TC002_LoginForNegative">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.click(SeMethods.java:135)
	at pages.LoginPage.clickLogInForFailer(LoginPage.java:48)
	at testcases.TC002_LoginForNegative.loginAndLogout(TC002_LoginForNegative.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="16.958" classname="testcases.TC001_LoginAndLogout">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.verifyPartialText(SeMethods.java:240)
	at pages.HomePage.verifyLoggedName(HomePage.java:30)
	at testcases.TC001_LoginAndLogout.loginAndLogout(TC001_LoginAndLogout.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="5.5" classname="testcases.TC002_LoginForNegative"/>
  <testcase name="loginAndLogout" time="7.572" classname="testcases.TC001_LoginAndLogout">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.verifyPartialText(SeMethods.java:240)
	at pages.HomePage.verifyLoggedName(HomePage.java:30)
	at testcases.TC001_LoginAndLogout.loginAndLogout(TC001_LoginAndLogout.java:30)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="4.258" classname="testcases.TC001_LoginAndLogout">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.click(SeMethods.java:135)
	at pages.LoginPage.clickLogIn(LoginPage.java:43)
	at testcases.TC001_LoginAndLogout.loginAndLogout(TC001_LoginAndLogout.java:29)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="loginAndLogout" time="5.802" classname="testcases.TC002_LoginForNegative">
    <failure type="java.lang.RuntimeException">
      <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:61)
	at utils.Reporter.reportStep(Reporter.java:71)
	at wdMethods.SeMethods.verifyPartialText(SeMethods.java:240)
	at pages.LoginPage.verifyErrorMsg(LoginPage.java:56)
	at testcases.TC002_LoginForNegative.loginAndLogout(TC002_LoginForNegative.java:31)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
... Removed 10 stack frames]]>
    </failure>
  </testcase> <!-- loginAndLogout -->
  <testcase name="@AfterTest afterTest" time="0.0" classname="wdMethods.ProjectMethods">
    <ignored/>
  </testcase> <!-- @AfterTest afterTest -->
  <testcase name="@BeforeSuite beforeSuite" time="0.0" classname="wdMethods.ProjectMethods">
    <ignored/>
  </testcase> <!-- @BeforeSuite beforeSuite -->
  <testcase name="@AfterSuite afterSuite" time="0.0" classname="wdMethods.ProjectMethods">
    <ignored/>
  </testcase> <!-- @AfterSuite afterSuite -->
</testsuite> <!-- Test -->
